README.md

```diff
@@ -38,7 +38,15 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
 The passcode will expire after one week. To generate a new passcode:
 
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
```

Comment:
I don't think we need two steps here. 

"The passcode will expire after one week. To generate a new passcode, run `twilio rtc:apps:video:deploy --authentication passcode --override`."